CNV_pipeline/wdl_scripts/CCLE_CNV_PiPeline.wdl

```diff
@@ -0,0 +1,919 @@
+      scatter_count = scatter_count,
+      intervals = calling_intervals,
+      ref_fasta = ref_fasta,
+      ref_dict = ref_dict,
+      ref_fai = ref_fasta_index,
+      gatk_docker = gatk_docker,
+      disk_space = round(additional_disk + ref_size)
+	}
+
+	String mybam = select_first([CramToBam.output_bam, GatherBamFiles.output_bam])
+	Float bam_size = size(mybam, "GB")
+
+	scatter (calling_interval in SplitIntervals.interval_files) {
+    call CNNTasks.RunHC4 {
+      input:
+        input_bam = mybam,
+        input_bam_index = select_first([CramToBam.output_bam_index, GatherBamFiles.output_bam_index]),
+        reference_fasta = ref_fasta,
+        reference_dict = ref_dict,
+        reference_fasta_index = ref_fasta_index,
+        output_prefix = output_prefix,
+        interval_list = calling_interval,
+        gatk_docker = gatk_docker,
+        gatk_override = gatk_override,
+        preemptible_attempts = preemptible_attempts,
+        extra_args = extra_args,
+        disk_space_gb = round(bam_size + ref_size + additional_disk)
+    }
+
+    call CNNTasks.CNNScoreVariants {
+      input:
+        input_vcf = RunHC4.raw_vcf,
+        input_vcf_index = RunHC4.raw_vcf_index,
+        bam_file = RunHC4.bamout,
+        bam_file_index = RunHC4.bamout_index,
+        architecture_json = architecture_json,
+        architecture_hd5 = architecture_hd5,
+        reference_fasta = ref_fasta,
+        tensor_type = tensor_type,
+        inference_batch_size = inference_batch_size,
+        transfer_batch_size = transfer_batch_size,
+        intra_op_threads = intra_op_threads,
+        inter_op_threads = inter_op_threads,
+        reference_dict = ref_dict,
+        reference_fasta_index = ref_fasta_index,
+        output_prefix = output_prefix,
+        interval_list = calling_interval,
+        gatk_override = gatk_override,
+        gatk_docker = gatk_docker,
+        preemptible_attempts = preemptible_attempts,
+        mem_gb = mem_gb,
+        disk_space_gb = round((bam_size/scatter_count) + ref_size + additional_disk)
+    }
+	}
+
+	call CNNTasks.MergeVCFs as MergeVCF_HC4 {
+    input:
+      input_vcfs = CNNScoreVariants.cnn_annotated_vcf,
+      output_prefix = output_prefix,
+      gatk_override = gatk_override,
+      preemptible_attempts = preemptible_attempts,
+      gatk_docker = gatk_docker,
+      disk_space_gb = additional_disk
+	}
+
+	call CNNTasks.FilterVariantTranches {
+    input:
+      input_vcf = MergeVCF_HC4.merged_vcf,
+      input_vcf_index = MergeVCF_HC4.merged_vcf_index,
+      resource_fofn = resource_fofn,
+      resource_fofn_index = resource_fofn_index,
+      output_prefix = output_prefix,
+      snp_tranches = snp_tranches,
+      indel_tranches = indel_tranches,
+      info_key = info_key,
+      gatk_override = gatk_override,
+      preemptible_attempts = preemptible_attempts,
+      gatk_docker = gatk_docker,
+      disk_space_gb = additional_disk
+	}
+
+	call CNNTasks.SamtoolsMergeBAMs {
```

Comment:
same here and a few other places

Reply:
I will continue working on this pipeline before merging then